Add some clarifications to the Clear Schema History setting (#7062)
segmentio/segment-docs: src/connections/sources/schema/schema-unique-limits.md

If you hit any of the limits or would like to clear out old events or properties, you can clear the Schema data from your Source Settings. In your Source, navigate to Settings, then Schema Configuration. Scroll down to the **Clear Schema History** setting.

> warning ""

> > The settings to clear Identify/Groups traits will **not** be available if a Tracking plan is connected to the Source.

Reviewer: ```suggestion
> You can't clear Identify/Group traits if your Source is connected to a Tracking Plan. 
```
